docs/design.md

```diff
@@ -28,6 +28,7 @@ There is a more detailed explanation in the following paragraphs.
 | *Name* field in namespace | *Name* gets set to 'pmem-csi' to achieve own vs. foreign marking | *Name* gets set to VolumeID, without attempting own vs. foreign marking  |
 |Minimum volume size| 4 MB                   | 1 GB (see also alignment adjustment below) |
 |Alignment requirements |LVM creation aligns size up to next 4MB boundary  |driver aligns  size up to next alignment boundary. The default alignment step is 1 GB. Device(s) in interleaved mode will require larger minimum as size has to be at least one alignment step. The possibly bigger alignment step is calculated as interleave-set-size multiplied by 1 GB |
+|Huge pages supported by `mkfs` options<sup>4</sup>| no| yes|
```

Comment:
Why mention "mkfs" here? Are the options different, or is the result different (huge pages work with one mode but not the other)?

Reply:
I planned to have it first as "Huge pages supported" but then I was thinking, for technically oriented reader, I could add few words and make the information much richer, i.e. what exactly enables that support. The diff is really only in mkfs options. Actually, if we want to be really precise, it is added mkfs options and added xfs_io call in xfs case. So in that sense, yes perhaps we should not be too specific here, a technical person will find that information if really wants

Reply:
Let's use "Huge pages supported" here. But looking at your explanation below, it doesn't really "explain" this difference. It just states that we observed it during testing.

So under different circumstances it might also work in direct mode (alignment perhaps?) - we simply don't know, do we?